.circleci/src/workflows/commit_main.yml

```diff
@@ -1,7 +1,9 @@
 when:
-  and:
-    - << pipeline.parameters.run_commit_main >>
-    - equal: [main, << pipeline.git.branch >>]
+  or:
```

Comment:
I would prefer if you undo this, as all our run_xx parameters are boolean and i would expect this to also be boolean, but I see it makes it easier to run this workflow on other branches, so I am also fine with you suggestion.

Reply:
Hmm.. currently this flag `run_commit_main` can only be used for skipping the `commit_main` workflow in case we are on branch `main`. As we probably never want to skip this workflow on main (?) it seems that this flag was somewhat useless before?

Is there a different possibility to run the workflow on other branches?

Reply:
You are right, the before solution did not make sense. We could introduce an other boolean param, that is default false, and does run the commit_Main workflow without checking the branch. But this is not really better than your solution :thinking:

Reply:
Hmm yeah, this seems to make it unclear which param to set. I'd say, we merge this until we come up with a better solution ? :D

Reply:
Yes fine :+1: